owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -256,12 +259,17 @@ class PassCodeActivity : AppCompatActivity() {
                 }
                 startActivity(intent)
             }
+            passCodeViewModel.resetNumberOfAttempts()
             finish()
         } else {
             showErrorAndRestart(
                 errorMessage = R.string.pass_code_wrong, headerMessage = getString(R.string.pass_code_enter_pass_code),
                 explanationVisibility = View.INVISIBLE
             )
+            passCodeViewModel.increaseNumberOfAttempts()
+            if (passCodeViewModel.getNumberOfAttempts() >= 3) {
```

Comment:
I would try to define these values in constant variables, it is easier to find them if we need to change them.